eng/common/scripts/git-diff-changes.ps1

```diff
@@ -1,36 +1,33 @@
+
```

Comment:
Let's name this script "get-changedfiles.ps1"

Reply:
Done.